app/services/api_tracking_information.rb

```diff
@@ -0,0 +1,39 @@
+class ApiTrackingInformation
+  require 'open-uri'
+  require 'pry'
```

Comment:
I believe this is accidentally included in the main code, this should be included in `development` group in Gemfile